cmd/juno/juno.go

```diff
@@ -2,13 +2,27 @@ package main
 
 import (
 	"fmt"
+	"runtime/debug"
 
 	"github.com/NethermindEth/juno/node"
 	"github.com/NethermindEth/juno/utils"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
+var version = "v0.1.0"
```

Comment:
We should not be using a string to maintain our version, it is prone to not being updated